test/functional/apps/visualize/input_control_vis/dynamic_options.js

```diff
@@ -34,13 +34,13 @@ export default function ({ getService, getPageObjects }) {
 
       it('should fetch new options when string field is filtered', async () => {
         const initialOptions = await comboBox.getOptionsList('listControlSelect0');
-        expect(initialOptions.trim().split('\n').join()).to.equal('BD,BR,CN,ID,IN,JP,NG,PK,RU,US');
+        expect(initialOptions.trim().split('\n').join()).to.equal('BD,BR,CN,ID,IN,JP,NG,PK,RU');
```

Comment:
> Changed underlying list virtualization library. Some EuiComboBox and EuiFilter* lists will technically have one fewer item in the DOM on initial load, although no real change occurs because the now-absent item was not visible initially without scrolling.